.changeset/little-guests-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hopper-ui/components": minor
```

Comment:
![image](https://github.com/user-attachments/assets/3a29c8b1-e916-4ca7-9cb4-9fb177d5e8b3)

Should we list Select.option since it's documented in the page and part of select?

Reply:
We did for Listbox

Reply:
I can remove it from the Select sub components if you think it's best? i have no preference either way..

Reply:
It's the first of this kind we got, we just need to decide, I think that in some cases there would be a lot of subcomponents listed.

Reply:
ok, I guess it's obvious so I'll remove it.